### ISSUE TITLE
Предложение

### ISSUE THREAD

**Issue by alliluja** [mil-ast/odo24#24]
Предлагаю при вводе нового обслуживания - проверять пробег, и если он больше уже установленного - обновлять.
![Screenshot 2021-06-13 175522](https://user-images.githubusercontent.com/35689026/121812498-a6192000-cc70-11eb-9503-64b0813a7a1b.png)

**Comment by mil-ast** [comment]
Согласен. Когда-то так и задумывалось, но, похоже, до реализации не дошло.
Встречный вопрос: нужно ли мобильное приложение и, если да, актуальна ли веб-версия будет? )))

**Comment by alliluja** [comment]
Думаю имеет смысл адаптировать сайт под мобильники, ставить еще одну приложуху - и так памяти в телефоне не хватает.
А использовать на мобильном намного удобнее, сейчас приходится собирать чеки, на них писать километраж и дома уже заносить в odo24

**Comment by mil-ast** [comment]
> Думаю имеет смысл адаптировать сайт под мобильники, ставить еще одну приложуху - и так памяти в телефоне не хватает.
> А использовать на мобильном намного удобнее, сейчас приходится собирать чеки, на них писать километраж и дома уже заносить в odo24

Сайт уже адаптирован. Кроме того, в хроме есть возможность создать иконку и тогда сайт будет открываться визуально как приложение.

**Comment by mil-ast** [comment]
По предложениям я сделаю, но позже. Времени катастрофически не хватает на проект. Но поддерживать буду и дальше, т.к. сам им пользуюсь )
